src/AppBanner/index.stories.tsx

```diff
@@ -40,18 +40,18 @@ export const defaultStory = (): ReactElement => {
         <StorybookWrapper>
             <BrazeMessageComponent
                 componentName={componentName}
-                logButtonClickWithBraze={(internalButtonId) => {
-                    console.log(`Button with internal ID ${internalButtonId} clicked`);
-                }}
-                submitComponentEvent={(componentEvent) => {
-                    console.log('submitComponentEvent called with: ', componentEvent);
-                }}
                 brazeMessageProps={{
                     header,
                     body,
                     imageUrl,
                     cta,
                 }}
+                logButtonClickWithBraze={(internalButtonId) => {
+                    console.log(`Button with internal ID ${internalButtonId} clicked`);
+                }}
+                submitComponentEvent={(componentEvent) => {
+                    console.log('submitComponentEvent called with: ', componentEvent);
+                }}
```

Comment:
I guess this moved around as we experimented with the interface to `BrazeMessageComponent`. Might be worth moving back to reduce the size of the diff on this PR? What do you reckon?

Reply:
Done